libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Identity/SFIdentityCoordinator.h

```diff
@@ -40,6 +40,16 @@ extern const NSTimeInterval kSFIdentityRequestDefaultTimeoutSeconds;
  */
 extern NSString * const kSFIdentityErrorDomain;
 
+/**
+ * The Authorization header value.
+ */
+extern NSString * const kHttpHeaderAuthorization;
```

Comment:
Revert this change since we're switching to `SFRestAPI`. We don't need to `extern` this.